pypesto/optimize/util.py

```diff
@@ -0,0 +1,70 @@
+from ..engine import Engine, SingleCoreEngine
+from ..objective import HistoryOptions
+from ..store.save_to_hdf5 import get_or_create_group
+from pathlib import Path
+
+
+import h5py
+
+
+def check_hdf5_mp(history_options: HistoryOptions,
+                  engine: Engine):
```

Comment:
```suggestion
def check_hdf5_mp(
    history_options: HistoryOptions,
    engine: Engine,
) -> Union[str, None]:
```
```suggestion
def check_hdf5_mp(history_options: HistoryOptions,
                  engine: Engine):
```

Reply:
dou you also see two suggestions? no idea what went wrong

Reply:
yes also see two suggestions. Guess, that you meant the one where something actually changes.